framework/src/main/java/org/checkerframework/common/basetype/messages.properties

```diff
@@ -31,6 +31,7 @@ invariant.cast.unsafe=cannot cast from "%s" to "%s"
 cast.unsafe.constructor.invocation=constructor invocation cast from "%s" to "%s" cannot be statically verified
 exception.parameter.invalid=invalid type in catch argument.%nfound   : %s%nrequired: %s
 throw.type.invalid=invalid type thrown.%nfound   : %s%nrequired: %s
```

Comment:
Is the old error key still needed?
Can we allow suppressions for the old key to continue to work for a bit?
Do add a changelog entry for this change.
We could do the `invalid` to `incompatible` change in a separate PR, together e.g. with the `exception.parameter.invalid` fixes... or more broadly reviewing how we name these messages.

Reply:
Yes, I undo the changes and will open a separate PR.